src/opencl/mod.rs

```diff
@@ -336,25 +417,23 @@ impl<T> KernelArgument<'_> for LocalBuffer<T> {
     fn push(&self, kernel: &mut Kernel) {
         kernel
             .builder
-            .arg_local::<u8>(self.length * std::mem::size_of::<T>());
+            .set_arg_local_buffer::<T>(self.length * std::mem::size_of::<T>());
     }
 }
 
-#[derive(Debug)]
+//#[derive(Debug)]
```

Comment:
can we get a manual debug impl?

Reply:
https://github.com/kenba/opencl3/issues/14#issuecomment-845172591 :)